src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -1,16 +1,38 @@
-import { makeStyles, Typography } from '@material-ui/core';
+import {
+  Grid,
+  IconButton,
+  makeStyles,
+  Tooltip,
+  Typography,
+} from '@material-ui/core';
+import { AddCircle } from '@material-ui/icons';
 import { Skeleton } from '@material-ui/lab';
 import clsx from 'clsx';
+import { Many } from 'lodash';
 import React from 'react';
 import { useParams } from 'react-router-dom';
+import { BooleanProperty } from '../../../components/BooleanProperty';
+import { DataButton } from '../../../components/DataButton';
+import { useDialog } from '../../../components/Dialog';
+import { useDateFormatter } from '../../../components/Formatters';
+import { PencilCircledIcon } from '../../../components/Icons';
```

Comment:
This is different than the component we use in other similar scenes. From Language Engagement Detail:

![image](https://user-images.githubusercontent.com/728777/96351788-c534c300-108b-11eb-9c17-e735a6efb76e.png)

Reply:
I actually copied this from Project Details:
<img width="1268" alt="Screen Shot 2020-10-18 at 11 26 08 AM" src="https://user-images.githubusercontent.com/43487134/96376591-c40ba080-1134-11eb-8531-c374ef9358ee.png">

I believe in language engagement details we have a breadcrumb to project and a link out to language because we have details scenes for those, but here we don't have a detail scene for Organization.